### PR TITLE
feat: create event card

### LINKED ISSUE #3: Criar card de evento
Criar card par exibição de informações sobre o evento seguindo os padrões de código do projeto.

Abaixo uma imagem par guiar o desenvolvimento.

Image

### PR DESCRIPTION
**Pull request by lucivandosousa**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new `EventCard` component for enhanced event display, showcasing details such as title, date, location, and description.
	- Added support for date formatting using the date-fns library for improved date presentation in the application.

- **Improvements**
	- Enhanced the user interface with a structured representation of events, ensuring a more readable and visually appealing layout.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@radix-ui/react-slot": "^1.0.2",
     "class-variance-authority": "^0.7.0",
     "lucide-react": "^0.365.0",
+    "moment": "^2.30.1",
```
Comment (cicerohen) [inline package.json]: No caso iremos usar essa.
https://date-fns.org/

File: src/app/page.tsx [hunk 2]
```diff
@@ -94,12 +95,13 @@ export default function Home() {
           <div className="grid gap-4 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4">
             {Array.from(Array(10).keys()).map((e) => {
               return (
-                <div
+                <EventCard
```
Comment (cicerohen) [inline src/app/page.tsx]: Pode ser usado o padrão composition sugerido no projeto

File: src/components/event-card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import "moment/locale/pt-br"
+
+interface EventCardProps {
+  title: string,
+  date: string,
+  locale: string,
+  description: string
+}
+
+export default function EventCard({title, date, locale, description}: EventCardProps) {
```
Comment (cicerohen) [inline src/components/event-card/index.tsx]: Pode ser usado o padrão composition sugerido no projeto

File: package.json (hunk 1 above)
Comment (lucivandosousa) [reply package.json]: A biblioteca foi adicionada e substituída no componente.

File: src/app/page.tsx (hunk 2 above)
Comment (lucivandosousa) [reply src/app/page.tsx]: O componente foi convertido para o padrão "Composition Pattern"

File: src/components/event-card/index.tsx (hunk 3 above)
Comment (lucivandosousa) [reply src/components/event-card/index.tsx]: O componente foi convertido para o padrão "Composition Pattern"

**Pull request merged**